### PR TITLE
Slack publish (was "npm auto publish")

### PR DESCRIPTION
**Pull request by jeremywiebe**
Linked PRs: N/A

## Changes
- ~~Sets up CircleCI to automatically publish to npm on merges to `master`~~
- Configures project so it can be released using Mobitron (https://github.com/mobify/mobitron/pull/124)
- Sets up doc linting on `README.md` and `RELEASE.md`

## How To Test
- Do a release and merge to `master`
- Monitor CircleCI build and ensure it publishes the package to npm

## Applicable Research Resources
- N/A

## TODOs:
- [ ] +1
- [x] Updated README

~~- [ ] Updated CHANGELOG~~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+## To be released
```
Comment (sap1ens) [inline CHANGELOG.md]: I'd avoid empty `To be released` section

Comment (jeremywiebe) [inline CHANGELOG.md]: @sap1ens  What's the reasoning for that?  I was thinking that if we have it there by default, people won't get it wrong and break the Mobitron release script.

Comment (sap1ens) [inline CHANGELOG.md]: True, but in this case if you try to release it, mobitron will release an empty changelog, which is not great. Anyway, I agree with your reasoning :)

Comment (jeremywiebe) [inline CHANGELOG.md]: @sap1ens  Maybe we should update Mobitron to check if `develop` is ahead of `master`.  If it is and the CHANGELOG is empty, we complain loudly and abort? (in a future PR :) )

Comment (sap1ens) [inline CHANGELOG.md]: Yep, sounds good :)

**Comment by jeremywiebe** [comment]
Slack 👍  from Slava. :)  

screen shot 2017-05-05 at 1 25 37 pm

**Pull request merged**